contents/blog/posthog-surveys-launched.md

```diff
@@ -0,0 +1,45 @@
+---
+date: 2023-10-19
+title: User surveys launch out of beta
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+featuredImage: ../images/blog/posthog-vs-amplitude/posthog-vs-amplitude.jpg
+featuredImageType: full
+author:
+  - joe-martin
+hideAnchor: true
+category: PostHog news
+---
+
+In June we launched the public beta of our user surveys tool, which in its first incarnation enabled you to ask simple questions with an open text field. Now, after many iterations and additions, the beta is over and we’re introducing new pricing for [PostHog surveys](/docs/surveys). 
+
+As with other PostHog tools there’s a generous free tier which enables everyone to get started and is suitable for small projects and MVPs. Beyond that, billing is based on usage via the number of responses you receive — no base rates, no additional fees.  
+
+The new **pricing kicks in on October 19th**, but we're also giving users who participated in the beta an enhanced plan with 1,000 free monthly responses to thank them for their support. We’ll also honor any annual contracts and users on these plans won’t pay for surveys until their contract renews. 
+
+| Responses per month | Price per response |
+| --- | --- |
+| 0-250 | Free |
+| 251-500 | $0.20 |
+| 501-1,000 | $0.10 |
+| 1,001-5,000 | $0.03 |
+| 5,001+ | $0.02 |
+
+We’ve prioritized surveys over the last few months to finally close the loop and turn PostHog into a truly all-in-one product stack. You can now gather user research, test changes, deploy them, observe behaviors, and measure results all in one place, in a way no other platform allows.
+
+Naturally, all the tools interconnect as well. Surveys linked to flags, results visible in insights and replays? PostHog does that.
+
+We’ve made sure that surveys are perfect for the most common use-cases, including [NPS surveys](/tutorials/nps-survey), customer satisfaction surveys, and Superhuman’s [product-market fit survey](/blog/measure-product-market-fit). However, there’s a lot more you can do beyond this, such as using the link question type to arrange user interviews!
+
+Here’s a quick summary of some of the work we’ve done, and major features we’ve added in the two months since the beta launched:
+
+- **[June 20th:](/changelog/2023#surveys-beta-released)** Beta launched with open text surveys
+- **[July 25th:](/changelog/2023#emoji-and-number-surveys-types-released)** Emoji and number rating survey types added
+- **[August 8th:](/changelog/2023#multiple-choice-surveys-released)** Multiple choice surveys added
+- **[August 18th:](/changelog/2023#wait-periods-now-available-for-surveys)** Wait periods and thank you messages added
+- **[September 14th:](/changelog/2023#automatic-nps-scoring-added-to-surveys)** Automatic NPS calculation added
```

Comment:
I think by then we'll also be able to add

- Survey popup redesigns
- Multiple questions
- Templates

(obvs TBD but just writing an extra reminder here)

Reply:
Yep, I'll update this before launch!